doc/de/Addons.md

```diff
@@ -191,184 +191,300 @@ Komplette Liste der Hook-Callbacks
 
 Eine komplette Liste aller Hook-Callbacks mit den zugehörigen Dateien (am 01-Apr-2018 generiert): Bitte schau in die Quellcodes für Details zu Hooks, die oben nicht dokumentiert sind.
 
-index.php:	Addon::callHooks('init_1');
-index.php:	Addon::callHooks('app_menu', $arr);
-index.php:	Addon::callHooks('page_content_top', $a->page['content']);
-index.php:	Addon::callHooks($a->module.'_mod_init', $placeholder);
-index.php:	Addon::callHooks($a->module.'_mod_init', $placeholder);
-index.php:	Addon::callHooks($a->module.'_mod_post', $_POST);
-index.php:	Addon::callHooks($a->module.'_mod_afterpost', $placeholder);
-index.php:	Addon::callHooks($a->module.'_mod_content', $arr);
-index.php:	Addon::callHooks($a->module.'_mod_aftercontent', $arr);
-index.php:	Addon::callHooks('page_end', $a->page['content']);
+### index.php
+
+    Addon::callHooks('init_1');
+    Addon::callHooks('app_menu', $arr);
+    Addon::callHooks('page_content_top', $a->page['content']);
+    Addon::callHooks($a->module.'_mod_init', $placeholder);
+    Addon::callHooks($a->module.'_mod_init', $placeholder);
+    Addon::callHooks($a->module.'_mod_post', $_POST);
+    Addon::callHooks($a->module.'_mod_afterpost', $placeholder);
+    Addon::callHooks($a->module.'_mod_content', $arr);
+    Addon::callHooks($a->module.'_mod_aftercontent', $arr);
+    Addon::callHooks('page_end', $a->page['content']);
+    
+### include/api.php
+
+	Addon::callHooks('logged_in', $a->user);
```

Comment:
The indentation style still are mixed, in one case it is a tab, in the other 4 spaces. Please choose one of the other for the whole file.

Reply:
I think it should be consistent even broader, for the DE and EN versions of the file alike.

Reply:
Fixed